### PR TITLE
Change Acceptance Criteria

### PR DESCRIPTION
**Pull request by Jiajia-Cui**

### CODE REVIEW

File: protocol/0034-PROB-prob_weighted_liquidity_measure.ipynb [hunk 1]
```diff
@@ -72,10 +72,10 @@
     "\n",
     "## Acceptance Criteria\n",
     "\n",
-    "1. It is always true that for $x_{min}$, $x_{max}$ (from risk model) and $S$ we work with (either `mid` or `last trade price` as appropriate) we have $x_{min} < S < x_{max}$. (<a name=\"0034-PROB-001\" href=\"#0034-PROB-001\">0034-PROB-001</a>)\n",
-    "2. If there is volume on either side of the order book within $[x_{min},x_{max}]$ then the reported liquidity is strictly more than $0$ when using the `log-normal` risk model (with any valid parameters). (<a name=\"0034-PROB-002\" href=\"#0034-PROB-002\">0034-PROB-002</a>)\n",
-    "3. If there is no mid price in continuous trading then the reported liquidity is $0$. (<a name=\"0034-PROB-003\" href=\"#0034-PROB-003\">0034-PROB-003</a>)\n",
-    "4. If there is no last trade price in an auction (opening auction in particular) then the reported liquidity is $0$. (<a name=\"0034-PROB-004\" href=\"#0034-PROB-004\">0034-PROB-004</a>)\n"
+       "1. Pegged Price and Volume from LP commitment should carry the correct amount\n",
```
Comment (davidsiska-vega) [inline protocol/0034-PROB-prob_weighted_liquidity_measure.ipynb:75]: What does this mean (ie if I want to test this how do I do it)?

File: protocol/0034-PROB-prob_weighted_liquidity_measure.ipynb [hunk 2]
```diff
@@ -72,10 +72,10 @@
...
+    "2. If there is volume on either side of the order book within $[x_{min},x_{max}]$ then the reported liquidity is strictly more than $0$ when using the `log-normal` risk model (with any valid parameters)\n",
+    "3. More details please see features tests: integration/features/liquidity-provisin \n",
```
Comment (davidsiska-vega) [inline protocol/0034-PROB-prob_weighted_liquidity_measure.ipynb:77]: The acceptance criteria shouldn't refer to anything in the `vega` repo; the `spec` repo should be self-contained.

File: qa-scenarios/Debug-Featuretest-BondSlashing.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (davidsiska-vega) [inline qa-scenarios/Debug-Featuretest-BondSlashing.json:1]: I don't think you want this as part of the PR, do you?

File: qa-scenarios/bond-slashing.feature.mock [hunk 4]
```diff
@@ -1,27 +1,128 @@
-Feature: Test liquidity provider bond slashing 
+# Test LP bond slashing
```
Comment (davidsiska-vega) [inline qa-scenarios/bond-slashing.feature.mock:1]: Does this run with current core version? If yes, please remove the `.mock` at the end.

**Review by davidsiska-vega — COMMENTED** [review]
I don't exactly follow why all the feature tests are submitted in a PR saying "change acceptance criteria". Are they part of the acceptance criteria? If yes could you clearly reference please?

**Comment by Jiajia-Cui** [comment]
This is a mess, let me start again from scratch.

**Pull request closed without merging**